### ISSUE TITLE
jobs: No such file or directory

### ISSUE THREAD

**Issue by Cherkah** [garabik/grc#182]
hi
when my terminal is open / new consol or instance i got:
`grc: jobs: No such file or directory`

because of this, i can't use jobs cmd!

my .bashrc's grc settings:
````
#! /usr/bin/env bash
...
....
#### GRC: Dynamic aliases
 for cmd in `ls /usr/share/grc/conf.*`; do
   cmd="${cmd##*conf.}"
   type "${cmd}" >/dev/null 2>&1 && alias "${cmd}"="$( which grc ) --colour=auto ${cmd}"
 done

#### Static aliases
#[[ -s "/etc/profile.d/grc.sh" ]] && source /etc/profile.d/grc.sh
````

i wase asking myself about automatic aliases for bash: does this line `[[ -s "/etc/profile.d/grc.sh" ]] && source /etc/default/grc.sh` is equivalent with the one you give:  `[[ -s "/etc/grc.sh" ]] && source /etc/grc.sh`?

+++++ 
Does this writing ` for cmd in `ls /usr/share/grc/conf.*`; do` is not more better than this one:
`for cmd in g++ gas head make ld ping6 tail traceroute6 $( ls /usr/share/grc/ ); do`

**Comment by lnxbil** [comment]
> when my terminal is open / new consol or instance i got: `grc: jobs: No such file or directory`
> 
> because of this, i can't use jobs cmd!

Yes, that's normal. `jobs` is a bash-builtin and therefore cannot be colorized with `grc jobs`. You can however colorize it with this:

```
$ jobs | grc -c conf.jobs cat /dev/stdin
[1]  + suspended  vim
```

It's a little bit clumsy, but it works.
